tst/CTA.WebForms2Blazor.Tests/Factories/FileConverterFactoryTests.cs

```diff
@@ -39,13 +40,13 @@ public void OneTimeSetup()
         [SetUp]
         public void Setup()
         {
-            var webFormsWorkspaceManager = new WorkspaceManagerService();
+            var webFormsProjectAnalyzer = new ProjectAnalyzer(_testProjectPath, null);
             var blazorWorkspaceManager = new WorkspaceManagerService();
 
             blazorWorkspaceManager.CreateSolutionFile();
-            webFormsWorkspaceManager.CreateSolutionFile();
+            //webFormsWorkspaceManager.CreateSolutionFile();
```

Comment:
remove unused code also, repo has history to keep track of everything if ever needed.